packages/ComponentLibrary/src/components/Widgets/TabWidget.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useCallback, useEffect, useMemo } from 'react';
+import { Box, Button } from '@mui/material';
+import FormView from '../FormView';
+import { theme } from '../../theme';
+import PencilIcon from '../../assets/icons/edit-2.svg';
+import SaveIcon from '../../assets/icons/save.svg';
+import { styles, sx } from './styles';
+import { useRecordContext } from '../../../../MainUI/src/hooks/useRecordContext.ts';
+import { TabWidgetProps } from './types.ts';
+
+const TabWidget: React.FC<TabWidgetProps> = ({
+  onSave,
+  onCancel,
+  editButtonLabel = '',
+  cancelButtonLabel = '',
+  saveButtonLabel = '',
+  noRecordText = '',
+}) => {
+  const { selectedRecord, setSelectedRecord } = useRecordContext();
```

Comment:
Would be possible to pass this via props, so we can avoid using the context directly in the component library?